src/lib/y2network/widgets/interfaces_table.rb

```diff
@@ -100,7 +100,7 @@ def note(interface, conn)
 
       # Constructs device description for inactive s390 devices
       def device_item(device)
-        [device.id, friendly_name(device), _("Not activated"), device.id, ""]
+        [device.id, description_for(device.interface), _("Not activated"), device.id, ""]
```

Comment:
@joseivanlopez @imobachgs but `description_for` expect who parameters and we giving only one, isnt it? so we need to also initialize by default conn to nil.

Reply:
@joseivanlopez @imobachgs at the end we are going to need a unit test as suggested in https://github.com/yast/yast-network/pull/1247#pullrequestreview-760685223 :stuck_out_tongue_winking_eye:

Reply:
Another problem is that only offline S390 Group Devices are listed, and therefore the interface is basically `nil`.